packages/core/ios/EmbraceManager.swift

```diff
@@ -36,6 +38,7 @@ class SDKConfig: NSObject {
         self.disableAutomaticViewCapture = from["disableAutomaticViewCapture"] as? Bool ?? false
         self.endpointBaseUrl = from["endpointBaseUrl"] as? String
         self.disableNetworkSpanForwarding = from["disableNetworkSpanForwarding"] as? Bool ?? false
+        self.ignoredURLs = from.value(forKey: "disabledUrlPatterns") as? [String] ?? []
```

Comment:
not sure which is preferred but I see the other lines doing `from["key"]`, could we do the same here or switch the other ones to match this style?

Reply:
ah, this is because I pulled that line from `packages/react-native-otlp/ios/RNEmbraceOTLP.swift` which is using `from.value`. will clean this to use `from["xxx"]` but I'll unify the behavior when I work on removing the code duplication

Reply:
done